### ISSUE TITLE
Mask filtering Script

### ISSUE THREAD

**Issue by zhh6425** [Pointcept/OpenIns3D#31]
Hi, Thanks for the great work!!

I have noticed the paper mentions a set of mask filtering processes for the MPM module. But I can't find the code related to that. How can I generate masks on my own datasets with the same mask filtering in the paper?

And also, will you provide the mask3d checkpoint with Mask scoring that you mention in the paper?

Thanks!

**Comment by zhuoyuan00** [comment]
Yes I did not find the PMP code either. It seems the authors provided the masks directly.

**Comment by ZheningHuang** [comment]
Dear Both,

Apologies for the delayed response—I was not very active due to submissions and being on annual leave.

The Mask Proposal Module (MPM) was inherently adapted from Mask3D. In the demo [here](https://github.com/Pointcept/OpenIns3D#zero-shot-inference-with-single-vocabulary), MPM is included to produce mask for our input pcd. 

Regarding the benchmark results, to ensure a fair comparison with other works and ease of use—since many methods at the time of OpenIns3D followed similar strategies for mask proposal—we provided mask proposals directly.

If you want to generate masks organically, I recommend using [Mask3D easy setup](https://github.com/cvg/Mask3D), as the mask results were originally obtained from there. Additionally, [Segment3D](https://github.com/LeapLabTHU/Segment3D) appears to be a stronger version of the class-agnostic mask proposal.

Zhening
